tests/Microsoft.Identity.Web.Test.LabInfrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -23,7 +24,14 @@ public KeyVaultSecretsProvider()
 
             };
             defaultcertloader.LoadCredentialsIfNeededAsync(credentialDescription).GetAwaiter().GetResult();
-            Certificate = credentialDescription.Certificate;
+            if (credentialDescription.Certificate == null)
```

Comment:
ℹ️ I struggled because the cert was not in the machine, but the user credentials store, so I thought I'd add a more explicit error